tests/components/growatt/test_sensor.py

```diff
@@ -0,0 +1,130 @@
+"""The tests for the Growatt platform."""
+import json
+from unittest import mock
+
+import pytest
+import requests_mock
+import growatt
+
+import homeassistant.components.growatt.sensor as victim
+
+
+@mock.patch("growatt.GrowattApi.login", return_value={"userId": "1"})
+def test_login(_):
+    """Test logging in."""
+    client = growatt.GrowattApi()
+    login_success = victim.login(client, "foo", "bar")
+    assert login_success
+
+
+def test_convert_multiplier():
+    """Test converting multipliers."""
+    assert (
+        victim.GrowattPlant(None, None, "", "").convert_multiplier(
```

Comment:
The convert method is static method, isn't it?

Reply:
indeed it is